deps/rabbitmq_stomp/BUILD.bazel

```diff
@@ -4,6 +4,7 @@ load("@rules_erlang//:dialyze.bzl", "dialyze", "plt")
 load(
     "//:rabbitmq.bzl",
     "BROKER_VERSION_REQUIREMENTS_ANY",
+    "ENABLE_FEATURE_MAYBE_EXPR",
```

Comment:
We will remove enabling the maybe feature at runtime for 3.13 as 3.13 requires OTP 26.
However, it's needed right now.